DigitalLearningSolutions.Data/Services/GroupsService.cs

```diff
@@ -418,7 +427,7 @@ int centreId
             if (groupCourse == null)
             {
                 transaction.Dispose();
-                logger.LogError("Attempted to add a course that a centre does not have access to to a group.");
+                logger.LogError("Attempted to add a course that a centre does not have access to a group.");

```

Comment:
Minor, but I think the "duplicated" `to` makes sense here.

Reply:
Stupid english